service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AmendedOrderEventHandler.java

```diff
@@ -45,43 +46,47 @@
     private final SendDocumentService sendDocumentService;
     private final OtherRecipientsInbox otherRecipientsInbox;
 
+    @Async
     @EventListener
-    public void notifyParties(final AmendedOrderEvent orderEvent) {
+    @SuppressWarnings("unchecked")
+    public void notifyDigitalRepresentatives(final AmendedOrderEvent orderEvent) {
         final CaseData caseData = orderEvent.getCaseData();
         final DocumentReference orderDocument = orderEvent.getAmendedDocument();
         final List<Element<Other>> selectedOthers = orderEvent.getSelectedOthers();
         final String orderType = orderEvent.getAmendedOrderType();
 
-        sendNotificationToLocalAuthorityAndDigitalRepresentatives(caseData, orderDocument, selectedOthers, orderType);
-        sendNotificationToEmailServedRepresentatives(caseData, orderDocument, selectedOthers, orderType);
-    }
-
-    @EventListener
-    public void sendOrderByPost(final AmendedOrderEvent orderEvent) {
-        final CaseData caseData = orderEvent.getCaseData();
-        final List<DocumentReference> documents = List.of(orderEvent.getAmendedDocument());
-        final String orderType = orderEvent.getAmendedOrderType();
-        final List<Element<Other>> selectedOthers = orderEvent.getSelectedOthers();
-
-        if (!orderType.equals(AmendedOrderType.STANDARD_DIRECTION_ORDER)) {
-            Set<Recipient> allRecipients = new LinkedHashSet<>(sendDocumentService.getStandardRecipients(caseData));
+        Set<String> digitalRepresentatives = representativesInbox.getEmailsByPreference(caseData, DIGITAL_SERVICE);
+        Set<String> digitalRecipientsOtherNotNotified = (Set<String>) otherRecipientsInbox.getNonSelectedRecipients(
+            DIGITAL_SERVICE, caseData, selectedOthers, element -> element.getValue().getEmail()
+        );
+        digitalRepresentatives.removeAll(digitalRecipientsOtherNotNotified);
 
-            allRecipients.removeAll(otherRecipientsInbox.getNonSelectedRecipients(POST, caseData, selectedOthers,
-                element -> element.getValue()));
-            allRecipients.addAll(otherRecipientsInbox.getSelectedRecipientsWithNoRepresentation(selectedOthers));
+        final NotifyData notifyData = amendedOrderEmailContentProvider.getNotifyData(caseData,
+            orderDocument, orderType);
 
-            sendDocumentService.sendDocuments(caseData, documents, new ArrayList<>(allRecipients));
+        if (!digitalRepresentatives.isEmpty() & !orderType.equals(AmendedOrderType
```

Comment:
vv minor I would flip the `!A.equals(B)` to be `!B.equals(A)`, has better null safety